pkgs/servers/akkoma/default.nix

```
@@ -0,0 +1,218 @@
+{ lib
+, beamPackages
+, fetchFromGitea, fetchFromGitHub, fetchFromGitLab
+, cmake, file, libxcrypt
+, writeText
+, nixosTests
+, ...
+}:
+
+beamPackages.mixRelease rec {
+  pname = "pleroma";
+  version = "3.5.0";
+
+  src = fetchFromGitea {
+    domain = "akkoma.dev";
+    owner = "AkkomaGang";
+    repo = "akkoma";
+    rev = "v${version}";
+    hash = "sha256-Apt+6nI4zOCyRb5msPt5UF9vyaendyaOjrYBMl0DqRY=";
+  };
+
+  postPatch = ''
+    # Correct version number and remove dependency on OS_Mon
+    sed -E -i \
+      -e 's/(version\(")\d+\.\d+\.\d+("\))/\1${version}\2/' \
```

```suggestion
      -e 's/(version\(")[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+("\))/\1${version}\2/' \
```

sed doesn't have the `\d` class, but does have `[[:digit:]]`:

```shell-session
$ echo 'version: version("3.5.10"),' | sed -E -e 's/(version\(")\d\.\d\.\d("\))/\1XXXX\2/'
version: version("3.5.10"),
$ echo 'version: version("3.5.10"),' | sed -E -e 's/(version\(")[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+("\))/\1XXXX\2/'
version: version("XXXX"),
```

---

I removed it altogether as the current releases all carry the correct version number.